R/options.R

```diff
@@ -90,6 +90,7 @@ pt_options <- function(...) {
 #' @param additional_scripts
 #' A character vector containing file paths to any additional scripts which should be included. These will be sourced with includeScript.
 #' @param logo_position Which side should the logo be on? Can be "left" or "right".
+#' @param on_session_ended_fun An optional function to execute when the Shiny session ends.
```

Comment:
Need to specify in the documentation that the function takes the `state` object. I think probably you should say it also takes `...` so that it's open to accepting new arguments in the future.